Source/NSEPSImageRep.m

```diff
@@ -103,12 +123,13 @@ - (void) prepareGState
   // This is for subclasses only
 }
 
-// Drawing the Image 
+// Override to draw the specified page...
 - (BOOL) draw
 {
-  [self notImplemented: _cmd];
-
   [self prepareGState];
+  
+  [_pageRep _premultiply];
```

Comment:
The premultiply already happens inside of the called draw method. This also means you may remove the local declaration of this method. This is also true for the other files where this is used.

Reply:
Removed